### PR TITLE
feat: remove orchestrator address

### PR DESCRIPTION
**Pull request by SweeXordious**
## Overview

Closes https://github.com/celestiaorg/celestia-app/issues/1329

Depends on a new release of  the cosmos-sdk.

## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -185,7 +185,7 @@ require (
 
 replace (
 	github.com/celestiaorg/nmt v0.13.0 => github.com/celestiaorg/nmt v0.12.0
-	github.com/cosmos/cosmos-sdk => github.com/celestiaorg/cosmos-sdk v1.7.0-sdk-v0.46.7
+	github.com/cosmos/cosmos-sdk => github.com/celestiaorg/cosmos-sdk v1.7.0-sdk-v0.46.7.0.20230202145553-0b25d300d771
```
Comment (SweeXordious) [inline go.mod:188]: [note for reviewers]
will be changed by the new cosmos-sdk release

**Review by evan-forbes — COMMENTED** [review]
noice, let's bump the sdk and :shipit:

**Comment by Bidon15** [comment]
Awesome! I wonder to which release this is planned to be included? 
As `test-infra` also needs adaptations on removing orchestrator address

**Comment by evan-forbes** [comment]
@SweeXordious we just bumbed to v1.8.0-sdk-0.46.7, mind updating here? we can also do in a follow up PR

**Review by evan-forbes — APPROVED** [review]

**Pull request merged**